docs/services/vision/detection.md

```diff
@@ -254,9 +248,8 @@ my_detector = VisionClient.from_robot(robot, "my_detector")
 
 detections = await my_detector.get_detections_from_camera(cam1)
 
-# If you need to get an image first and then run detections on it,
-# you can do it this way (generally slower but useful if you need to
-# use the image afterwards):
+# If you need to use the image based on the detections, get the image first
```

Comment:
```suggestion
# If you need to save the image, get the image first
```
[opt] Not sure how to reword (I don't love my suggestion), but I think it's kinda unclear this way

Reply:
What I mean here is that maybe you just need to use the image period, regardless of the detections.